elementary/operations/upload_source_freshness.py

```diff
@@ -20,20 +20,23 @@ def run(self):
             raise click.ClickException(
                 "Path to dbt project is missing. Please run the command with `--project-dir <DBT_PROJECT_DIR>`."
             )
-        results = self.get_results()
-        self.upload_results(results)
-        click.echo("Uploaded source freshness results successfully.")
+        sources_file_contents = self.get_sources_file_contents()
+        if sources_file_contents:
```

Comment:
How could it be falsely?

Reply:
empty file, which should not happen, but since in the next line accessing to keys prefer checking it

Reply:
Personally I would just remove it as this is not something that can happen and this flow is a bit unclear.
If you want to keep it, maybe do:
```py
if not sources_file_contents:
    raise raise click.ClickException("Unexpectedly found an empty source.json file.")
```